code/include/playfab/QoS/RegionResult.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <string>
+#include <playfab/PlayFabMultiplayerDataModels.h>
+
+namespace PlayFab
+{
+    namespace QoS
+    {
+        /// <summary>
+        /// The result of pinging a datacenter.
+        /// </summary>
+        class RegionResult
+        {
+        public:
+            RegionResult(PlayFab::MultiplayerModels::AzureRegion region, int latencyMs, int errorCode);
+            RegionResult() = delete;
+
+            // The datacenter region
+            PlayFab::MultiplayerModels::AzureRegion region;
+
+            // Average latency to reach the data center
+            int latencyMs;
+
+            // Last error code recieved while pinging
+            int errorCode;
```

Comment:
if all the members are public, should this just be a struct? Or should these members be private with public getters/setters?

Reply:
Change to struct in this case